### ISSUE TITLE
Generate words

### ISSUE THREAD

**Issue by shantanuo** [blatinier/pyhunspell#57]
Is there any method to generate words similar to unmunch or wordforms?
For e.g. 

$ wordforms en_US.aff en_US.dic educate
educating
educated
educate
educates
educates

**Comment by shantanuo** [comment]
looks like "generate" method can do this. But can you show an example how to use it?

**Comment by shantanuo** [comment]
As per the unit test, this command should return boys, but it is returning an empty list.
hobj.generate('boy', 'girls')

**Comment by blatinier** [comment]
Yes I have the same problem myself, so does Travis I think. On Archlinux it was working great but since I switch to Debian this is not working anymore. Unfortunatly I did not had time to investigate.

My plan for this was:
 - Test archlinux hunspell dic & aff files on my debian
 - Test using python 2 & 3 (maybe a regression due to the C/C++ port & python3 support. I don't remember when this problem appeared
 - Hope for point 1 & 2 to work and/or get new ideas

**Comment by sarschu** [comment]
Any news on this topic? I am using generate on Ubuntu and the list is empty as described above.
